tests/cupy_tests/functional_tests/test_vectorize.py

```diff
@@ -35,13 +34,13 @@ def my_add(x, y):
 
         return self._run(my_add, xp, [dtype1, dtype2])
 
-    @testing.for_all_dtypes_combination(names=('dtype1', 'dtype2'))
-    @testing.numpy_cupy_array_equal(accept_error=TypeError)
-    def test_vectorize_sub(self, xp, dtype1, dtype2):
+    @testing.for_dtypes('bhilqefdFD')
```

Comment:
Talked with @asi1024 offline, this shrinks the test axis (combination of different dtypes) but it is acceptable as it is tested in `test_vectorize_add`.